packages/cozy-client/src/associations/HasManyFiles.js

```diff
@@ -3,12 +3,27 @@ import HasMany from './HasMany'
 import { QueryDefinition, Mutations } from '../queries/dsl'
 
 export default class HasManyFiles extends HasMany {
-  async fetchMore() {
-    const skip = this.getRelationship().data.length
+  async fetchMore(withCursor) {
```

Comment:
Using an options.cursor at true will be more readable IMHO , `fetchMore(true)` will not be very explicit. `fetchMore({ cursor: true })` will be easier to understand.

Reply:
👍 with Patrick